lib/KTextbox/__tests__/components/KTextboxVisualTest.vue

```diff
@@ -0,0 +1,67 @@
+<template>
+
+  <div>
+    <p>Label:</p>
+    <WithLabel />
+
+    <p>Validation:</p>
+    <Validation />
+
+    <p>Character Limit:</p>
+    <CharacterLimit />
+
+    <p>Disabled:</p>
+    <Disabled />
+
+    <p>Readonly:</p>
+    <Readonly />
+
+    <p>Number input:</p>
+    <Number />
+
+    <p>Password input:</p>
+    <Password />
+
+    <p>Textarea:</p>
+    <AsTextarea />
+
+    <p>Clearable:</p>
+    <Clearable />
+
+    <p>Invalid Text:</p>
+    <InvalidText />
+  </div>
+
+</template>
+
+
+<script>
+
+  import WithLabel from '../../../../docs/examples/KTextbox/WithLabel';
```

Comment:
Could these paths be changed as `~/examples/KTextbox/WithLabel` for better readability?

Reply:
Yes, I think that'd be better, and we should probably actually move those example files higher, to make clear they're not only related to docs. I think that'd be best done in a separate PR. Can we do it together with other visual testing improvements we're chatting about on Slack?

Reply:
That'd work for me :)